core/src/epicli/data/common/ansible/playbooks/roles/applications/templates/pgpool/01-secrets.yml.j2

```
@@ -11,3 +11,4 @@ data:
   pgpool_admin_password: "{{ data.pgpool.secrets.pgpool_admin_password | b64encode }}"
   pgpool_postgres_password: "{{ data.pgpool.secrets.pgpool_postgres_password | b64encode }}"
   pgpool_sr_check_password: "{{ data.pgpool.secrets.pgpool_sr_check_password | b64encode }}"
+  pgpool_sha_encryption_key: "{{ data.pgpool.secrets.pgpool_sha_encryption_key | b64encode }}"
```

```suggestion
  {% if data.pgpool.secrets.pgpool_sha_encryption_key is defined %}
    pgpool_sha_encryption_key: "{{ data.pgpool.secrets.pgpool_sha_encryption_key | b64encode }}"
  {% endif %}
```
If data.pgpool.secrets.pgpool_sha_encryption_key will be optional, shouldn't we add condition here?
I think we can also use this one if we are sourcing it: postgresql_role_config.specification.configuration.password_encryption
would be nice to keep it consistent